packages/babel-plugin/src/transform.ts

```diff
@@ -3,6 +3,7 @@ import { compile } from "@kuma-ui/compiler";
 import pluin from ".";
 import { sheet } from "@kuma-ui/sheet";
 
+// eslint-disable-next-line @typescript-eslint/require-await -- FIXME
```

Comment:
Example explanation: now this function does not include any `await` so ESLint is pointing out that :)